app/assets/stylesheets/govuk_publishing_components/components/_modal-dialogue.scss

```diff
@@ -0,0 +1,138 @@
+$govuk-modal-margin: govuk-spacing(6);
+$govuk-modal-close-button-size: 40px;
```

Comment:
Is this big enough for touch? I vaguely remember WCAG saying it should be 44px.

Reply:
Oh good spot there, yes, it should be 44x44

Reply:
Oh.. the button used to have a 2px border which is now gone, that's how I ended up with this. Will update.

Reply:
Updated addressing the close button size too. This should be ready for a final review.
As a reminder, the component is marked as experimental as it's likely to be updated based on user research, but it's a good way of sharing it between projects.